subprojects/docs/src/samples/webApplication/quickstart/build.gradle

```diff
@@ -1,19 +1,24 @@
+// START SNIPPET use-gretty-plugin
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'org.akhikhl.gretty:gretty:1.4.2'
+    }
+}
+apply plugin: 'org.akhikhl.gretty'
+// END SNIPPET use-gretty-plugin
+
 // START SNIPPET use-war-plugin
 apply plugin: 'war'
 // END SNIPPET use-war-plugin
-// START SNIPPET use-jetty-plugin
-apply plugin: 'jetty'
-// END SNIPPET use-jetty-plugin
 
 repositories {
-    mavenCentral()
+    jcenter()
 }
 
 dependencies {
     compile group: 'commons-io', name: 'commons-io', version: '1.4'
     compile group: 'log4j', name: 'log4j', version: '1.2.15', ext: 'jar'
 }
-
-httpPort = 8080
```

Comment:
Looks like we also want to show the configuration of the HTTP port. Something we can configure in Gretty as well.